src/chapter9a/Person.java

```diff
@@ -0,0 +1,35 @@
+package chapter9a;
+
+public class Person {
+    String name;
+    int age;
+    String gender;
+
+    public Person(String name) {
+        System.out.println("This is the second constructor. The name is: ");
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
```

Comment:
when you create a setter it is always good to add some validation rules. If(age < 0) then sout (Invalid Data) else this.age = age.